video_frame_metadata_registry.src.html

```diff
@@ -0,0 +1,61 @@
+<pre class='metadata'>
+Title: WebCodecs VideoFrame Metadata Registry
+Repository: w3c/webcodecs
+Status: NOTE-ED
+Shortname: webcodecs-videoframemetadata-registry
+Level: none
+Group: mediawg
+ED: https://w3c.github.io/webcodecs/videoframemetadata_registry.html
+TR: https://www.w3.org/TR/webcodecs-videoframemetadata-registry/
+Editor:
+
+Boilerplate: omit conformance
+
+Abstract: This registry is intended to enumerate the metadata fields that can be attached
+    to {{VideoFrame}} objects via the {{VideoFrameMetadata}} dictionary..
+
+Markup Shorthands:css no, markdown yes, dfn yes
+!Participate: <a href="https://github.com/w3c/webcodecs">Git Repository.</a>
+!Participate: <a href="https://github.com/w3c/webcodecs/issues/new">File an issue.</a>
+!Version History: <a href="https://github.com/w3c/webcodecs/commits">https://github.com/w3c/webcodecs/commits</a>
+</pre>
+
+<pre class='anchors'>
+spec: WEBCODECS; urlPrefix: https://w3c.github.io/webcodecs/#
+    type: dictionary
+        text: VideoFrame; url: dictdef-videoframe
+</pre>
+
+
+Registration Entry Requirements {#registration-entry-requirements}
+==================================================================
+
+A registration entry, called metadata, has the following requirements:
+1. Each metadata is defined as a separate {{VideoFrameMetadata}} dictionary member.
+2. Each metadata MUST be serializable.
+3. Each metadata should have clearly defined semantics.
+   In particular, its interactions with the media processing pipeline
+   (encoders, decoders, renderers...) should be well defined.
+
```

Comment:
Yes. For example, it seems sensible to require a W3C specification, and probably one that has been adopted by a WG.  If there is some expectation of changed behavior in WebCodecs (e.g. an encoder making use of FaceDetection metadata), then the registration should probably be approved by the MEDIA WG.

Reply:
I agree, although your comment suggests that another WG could update the registry without needing approval from Media WG, if there weren't any expectations on WebCodecs. It would mean coordinating between multiple WGs, but would we not still want Media WG approval, as publishers of the registry?

Reply:
I added the lines from the codec registry.

Reply:
@chrisn By default, it would probably make sense to require MEDIA WG approval for all "top level" entries in the registry. If it chose to do so, the MEDIA WG could delegate authority for a particular "top level" entry (e.g. a dictionary) to another entity.  However, before a registry entry is reviewed by MEDIA WG, there probably needs to be some minimum requirements.  For example, that the registry entry refer to a specification that has achieved some level of consensus. I also think we need to talk about what the requirements are for those submitting registration requests. Do they have to be participants in the W3C, so as to pass the IPR check?